Code/Framework/AzNetworking/AzNetworking/UdpTransport/UdpNetworkInterface.h

```diff
@@ -55,8 +55,11 @@ namespace AzNetworking
     //! ### Fragmentation
     //! 
     //! If the raw packet size exceeds the configured maximum transmission unit (MTU) then the packet is broken into
-    //! multiple reliable fragments to avoid fragmentation at the routing level. Fragments are always reliable so the original
-    //! packet can be reconstructed. Operations that alter the payload generally follow this step so that they can be
+    //! multiple fragments to avoid fragmentation at the routing level. Fragment reliability is controlled by the cvar
```

Comment:
It might be worth making this more of a warning to users regarding the usage of this cvar.. In effect, it will cause any packets that happen to exceed MTU ignore the request to send unreliably with no retransmit, and will force all packets over MTU to be reliable. Combined with MTU path discoverability this could be really surprising to users.

Reply:
I've detailed the functionality a bit more and added a separate warning/recommendation on usage.